containers/widget-server/README.md

```diff
@@ -37,9 +32,10 @@ This container works best with widgets that are build in the latest Create React
    </html>
    ```
 
-3. Make sure that your build artifacts are in the `static/` folder of your build directory have a `hash` in its name. This is the default for CRA.
+2. Depending on the libraries in use it may be necessary to pass the CSP nonce to them.
+   See the [example widget's index page](../../example-widget-mui/index.html) setting it up for webpack.
```

Comment:
wait i thought we are using vite now, not webpack?

Reply:
removed mention of webpack